cli/run/import-ts-plugin.ts

```diff
@@ -0,0 +1,13 @@
+import { resolve } from 'path';
+
+export default async (configPath: string) => {
+	try {
+		return (
+			await import(resolve(configPath, '../', 'node_modules/@rollup/plugin-typescript'))
```

Comment:
Ah, the problem with the dynamic import is that you are constructing a URL here dynamically. This will indeed not work. If you just write `import('@rollup/plugin-typescript'` here it will work, though, as Node will take care of resolving it correctly at runtime.
And this will also pick the plugin from the right directory as Rollup injects the compiled config file into the Node module loader mechanism where the original config would have been.

But with the latest comments, maybe this will become obsolete anyway.